test-packages/framework/src/framework.ts

```diff
@@ -1,19 +1,24 @@
 import { Position, Range } from "vscode-languageserver-types";
+import { TextDocumentPositionParams } from "vscode-languageserver";
+import { TextDocument } from "vscode-languageserver-textdocument";
 import { join } from "path";
 import { readFile, writeFile } from "fs/promises";
 import { DocumentCstNode, parse } from "@xml-tools/parser";
 import { buildAst } from "@xml-tools/ast";
 import { URI } from "vscode-uri";
 import {
   npmInstall,
-  deleteCopy,
+  deleteProject,
   createCopy,
   fileExitsSync,
   print,
   removeProjectContent,
 } from "./utils";
 
 import { TestFrameworkAPI, ProjectInfo, Config, ReadFileResult } from "./types";
+import { repeat } from "lodash";
```

Comment:
please maintain dependency in package.json.

Reply:
What do you mean? lodash is already in dependencies